ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -3913,18 +4010,58 @@ module VBD = struct
           raise (Xenopsd_error (Device_detach_rejected ("VBD", id_of vbd, s)))
```

Comment:
Hmm "Device_detach_rejected" doesn't make sense now. AFAICT this exception would be raised in clean_shutdown above, so it's being caught in the right place but now it is being called with VBD.deactivate. I believe we agreed the clean_shutdown etc. had to happen before deactivate, rather than attach. Would it make sense to rename this exception to e.g. Device_deactivate_rejected or do we need to do something else?

Reply:
That makes sense, so we can leave this as is?

Reply:
Yes.